docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx

```diff
@@ -27,9 +27,19 @@ Agent tokens are used to authenticate [Hybrid agents](/dagster-cloud/deployment/
 
 After the token is created:
 
-- **To edit a token's description**, click the **pencil icon**.
 - **To view a token**, click **Reveal token**. Clicking on the token value will copy it to the clipboard.
-- **To revoke a token**, click **Revoke**.
+- **To edit a token's description**, click the **pencil icon**.
+
+### Modifying agent tokens
+
+1. Sign in to your Dagster Cloud account.
+2. Click the **user menu (your icon) > Organization Settings**.
+3. Click the **Tokens** tab.
+4. Click **Edit** next to the agent token you'd like to change.
+
+The permissions dialog allows you to edit a token's ability to access certain deployments. By default, agent tokens have permission to access any deployment in the organization including branch deployments. This is called "Org Agent" and is set using the toggle in the top right of the dialog. To edit individual deployment permissions, "Org Agent" has to first be toggled off.
+
+- **To revoke a token**, click **Revoke** in the bottom left of the permissions dialog.
```

Comment:
This bit about revoking gets kinda lost down here now. I'm not sure how often this comes up, but we can add a section about this to make it easier to find:

```suggestion
### Revoking agent tokens

To revoke a token:

1. Sign in to your Dagster Cloud account.
2. Click the **user menu (your icon) > Organization Settings**.
3. Click the **Tokens** tab.
4. Click **Edit** next to the agent token you'd like to change.
5. Click **Revoke** in the bottom left of the permissions dialog. When prompted, confirm to proceed with revoking the token.
```